raiden/transfer/state_change.py

```diff
@@ -43,3 +43,29 @@ class ActionCancelTransfer(StateChange):
 
     def __init__(self, identifier):
         self.identifier = identifier
+
+
+class ActionTransferDirect(StateChange):
+    def __init__(self,
```

Comment:
Can you put all the arguments in the next line, like we have in the style guide?